husarion_ugv_docking/include/husarion_ugv_docking/send_to_dock_node.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2025 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HUSARION_UGV_DOCKING_HUSARION_UGV_DOCKING_SEND_TO_DOCK_NODE_HPP_
+#define HUSARION_UGV_DOCKING_HUSARION_UGV_DOCKING_SEND_TO_DOCK_NODE_HPP_
+
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+
+#include <nav2_msgs/action/dock_robot.hpp>
+#include <std_srvs/srv/set_bool.hpp>
+
+namespace send_to_dock {
```

Comment:
Use namespace with the name of the package:
```suggestion
namespace husarion_ugv_docking {
```